ui/views_test.py

```diff
@@ -37,3 +40,30 @@ def test_upload_admin(admin_client, mocker):
     assert response.status_code == 200
     assert response.context_data['dropbox_key'] == 'dbkey'
     assert response.template_name == ['ui/upload.html']
+
+
+def test_video_detail_hls(admin_client, video, videofileHLS, mocker):  # pylint: disable=unused-argument
+    """Test video detail page when HLS videofile is available"""
+    mocker.patch('ui.models.make_cloudfront_signed_url', return_value=videofileHLS.cloudfront_url)
+    video.status = 'Complete'
+    url = '/videos/{}/'.format(video.id)
+    response = admin_client.get(url)
+    assert 'videofile' in response.context_data
+    assert response.context_data is not None
+    js_settings_json = json.loads(response.context_data['js_settings_json'])
+    assert 'videofile' in js_settings_json
+    assert settings.VIDEO_CLOUDFRONT_DIST in js_settings_json['videofile']
```

Comment:
I think this should be `==` instead of `in`, since the values should always be exactly the same